clap_generate/examples/multicall.rs

```diff
@@ -0,0 +1,18 @@
+use clap::{App, AppSettings};
+use clap_generate::{generate, generators::Bash};
+use std::io;
+
+fn main() {
+    let mut app = App::new(env!("CARGO_CRATE_NAME"))
+        .setting(AppSettings::Multicall)
+        .subcommand(
+            App::new("test")
+                .subcommand(App::new("config"))
+                .subcommand(App::new("generate-completions")),
+        )
+        .subcommand(App::new("hello"));
+    for applet in app.get_subcommands_mut() {
+        let name = applet.get_name().to_owned();
+        generate(Bash, applet, &name, &mut io::stdout());
+    }
```

Comment:
If we are keeping this as an example, we probably want to
- Make this hostname or busybox style
- should we also generate completions for the base command (if busybox style)
- Only generate completions when requested
- Probably should show case `Shell`, rather than hard coding bash
- Add a `trycmd` test case for it (needs `trycmd` as a dependency, needs its own `tests/examples.rs`, needs a `.md` or `.toml` / `.stdout` / `.stderr` file

Reply:
I think I'd prefer to drop the completion generation example for now. The scope of examples changed between my initially writing it and it being reviewed, so it might be better to start from scratch.